pkg/build/pipeline.go

```diff
@@ -339,3 +340,6 @@ func (p *Pipeline) ApplyNeeds(ctx *PipelineContext) error {
 
 	return nil
 }
+
+//go:embed pipelines/*
+var f embed.FS
```

Comment:
Was this moved on purpose?

Reply:
Yes, it was causing my editor to mishighlight due to the `/*`